documentation/3_Layers_of_the_Reference_Architecture_Model/3_4_Process_Layer/3_4_4_Exchanging_Data.md

```diff
@@ -63,6 +63,32 @@ to switch from data pulling to data pushing easily. The sequence is depicted in
 
 ##### Figure 3.4.4.2: Out-of-band Data Exchange
 
+#### Semantic Interoperability
+
+Semantic Interoperability is crucial for Data Exchange in Data Spaces. This starts dhring
+Design-time, as described in the [Data Offering process](./3_4_2_Data_Offering.md#data-offering),
+and in the process of [publishing](./3_4_2_Data_Offering.md#data-provider-registering-self-descriptions)
+and [quering](./3_4_2_Data_Offering.md#data-consumer-searching-for-self-descriptions) self-descriptions.
+In the process of Data Exchange the Vocabulary Hub and the Vocabularies are invoked for semantic
+interoperability.
+
+The following steps are followed:
+
+1. Prior to the invokation of a data operation (in a synchronous or asynchronous way) by the data consumer one or multiple vocabularies
```

Comment:
Publish vocabulary is an optional step, might also use existing vocabularies, Link to data offering section.

Reply:
fixed